doc/source/operations/control-plane-operation.rst

```diff
@@ -201,6 +223,24 @@ following order:
 * Shut down seed VM
 * Shut down Ansible control host
 
+Full startup
+-------------
+
+If the entire control plane is powered down, it is best to bring the nodes up
+in the reverse order of shutdown:
+
+* Power on Ansible control host
+* Power on seed VM (and other service VMs)
+* Power on Ceph nodes (if applicable)
+   * Where possible, start the nodes running MON services first.
+   * Be sure to unset the ``noout`` flag, so that the cluster can
```

Comment:
noout is not relevant to rebalancing at all

Reply:
What does it actually do? The docs Ceph docs say:

> when you stop OSDs for maintenance, you might want to prevent CRUSH from automatically rebalancing the cluster. To avert this rebalancing behavior, set the cluster to noout

But also there's a warning after that which seems to say you don't need to do it since Luminous, so I'm confused

https://docs.ceph.com/en/reef/rados/troubleshooting/troubleshooting-osd/#stopping-without-rebalancing

Reply:
`noout` is telling Ceph not to treat the osds as being out of the cluster. This I suppose indirectly stops the rebalancing that would otherwise occur when an osd is out. 

There's also a `norebalance` flag that explicitly stops rebalancing, I don't know if we'd _need_ to set both, but I usually do so. 

@cityofships what would you suggest?